apps/test/unit/templates/sectionProgress/VirtualizedDetailViewTest.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import {UnconnectedVirtualizedDetailView} from '@cdo/apps/templates/sectionProgress/VirtualizedDetailView';
+import sectionProgress from '@cdo/apps/templates/sectionProgress/sectionProgressRedux';
+import {getStore, registerReducers, stubRedux, restoreRedux} from '@cdo/apps/redux';
+import {Provider} from 'react-redux';
+
+const studentData = [
+  {id: 1, name: 'studentb'},
+  {id: 3, name: 'studentc'},
+  {id: 0, name: 'studenta'}
+];
+
+describe('VirtualizedSummaryView', () => {
+  let DEFAULT_PROPS;
```

Comment:
nit: only use upper case for `const`s: https://github.com/airbnb/javascript#naming--uppercase